KalturaWowzaServer/release/register_media_server.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+require_once('../app/infra/general/BaseEnum.php');
```

Comment:
This is a client side script and as such it shouldn't use any kaltura object except for those defined in the client lib.
Also,
assume that it will use its own client lib and not the one that generated under /opt/kaltura/app that used by the html5 player.